conf/pxe_cluster/grub.cfg

```diff
@@ -0,0 +1,5 @@
+menuentry "Install Ubuntu" {
```

Comment:
What about adding a line or two of comments at the top of the file.

# This grub.cfg is use on UEFI based servers...

Reply:
Do we need the license whatnot?

Reply:
Added a simple comment header.

Reply:
Please add the license.
```
# Copyright 2017 ARICENT HOLDINGS LUXEMBOURG SARL and Cable Television Laboratories, Inc.
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#    http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```